Makefile

```diff
@@ -53,8 +53,12 @@ ifeq (${DBUILD_SITE_URL}, )
 endif
 
 
+# Set the path to the Chart.yaml file
+ROOT_DIR:=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+CHART_YAML:=${ROOT_DIR}/deploy/helm/charts/Chart.yaml
+
 ifeq (${IMAGE_TAG}, )
-  IMAGE_TAG = ci
+  IMAGE_TAG := ${shell yq e '.version' $(CHART_YAML)}
```

Comment:
Well, as we decided appVersion and chart version would now be tied, shouldn't matter. Also why do you think it would be an issue to add yq in the makefile? We are anyhow using yq in other script. All workflows will now work this way.

Reply:
Also, using yq is less prone to errors I feel.

Reply:
Because, a common workflow is... change code, then build the image with your changes. `yq` is not a common tool to have around. It should require `go` and `docker buildx`, kind of odd to require `yq`.

You're right, that this isn't exact, but close enough, I feel.

Reply:
Alright.